published/006-2016-March-7.md

```diff
@@ -0,0 +1,157 @@
+
+#### [**libp2p**](https://github.com/diasdavid/js-libp2p)
+
+@diasdavid made two new modules for stream muxing: [libp2p-spdy](https://github.com/diasdavid/js-libp2p-spdy) and [libp2p-multiplex](https://github.com/diasdavid/js-libp2p-multiplex). As well, [libp2p-swarm](https://github.com/diasdavid/js-libp2p-swarm) has a new API, more tests, less magic, and more flexibility. There is an open PR to track these changes; if you'd like to get involved, [follow it](https://github.com/diasdavid/js-libp2p-swarm/pull/20). Or, join in by checking out the latest [captain.log](https://github.com/ipfs/js-ipfs/issues/30#issuecomment-187950929).
+
+#### **ipfs-pad**
+
+We want to build an Etherpad-like product on top of IPFS. There's a lot of
+technical ground to cover to do this: how do we know how to sort edits? How do
+we deal with high latencies (days/weeks), or with simultaneous edits? How is the
+data transported? @noffle has been spearheading the process behind building
+this, and this week made a host of modules to resolve some of its dependencies:
+
+- [bisecting-numbers](https://github.com/noffle/bisecting-numbers) -
+  Integer-like number system where any number can be bisected to form infinite
+  integer subsystems.
+- [bisecting-between](https://github.com/noffle/bisecting-between) - Produces a
+  unique value that sorts between two other given values.
+- [hyperswarm](https://github.com/noffle/hyperswarm) - Create a p2p webrtc swarm
+  around a [hyperlog](https://github.com/mafintosh/hyperlog).
+
+(He also wrote a little [README
+generator](https://github.com/noffle/generate-readme) to help him out with his
+JS module READMEs.)
+
+#### **pubsub**
+
+@noffle also published a couple of early mad science modules that enable a
+peer-to-peer overlay network for gossip-based message exchange. These aren't
+strictly built for IPFS, but are an experimental shim to enable projects like
+@haad's [orbit-db](https://github.com/haadcode/orbit-db) to operate without a
+central server for message exchange between peers.
+
+- [secure-gossip](https://github.com/noffle/secure-gossip) - Secure, transport
+  agnostic, message gossip protocol. Any peer in the network can publish
+  messages, which will eventually propagate to the entire network via rounds of
+  gossip between each node's peers.
+- [pubsub-swarm](https://github.com/noffle/pubsub-swarm) - Form a p2p swarm of
+  nodes around a topic and exchange messages, with a traditional pubsub API.
+
+#### [**go-ipfs**](//github.com/ipfs/go-ipfs)
+
+@whyrusleeping set up [Teamcity](https://www.jetbrains.com/teamcity/). This cuts down on the long waits for Travis to run, and hopefully will mean faster CI tests. 
+Teamcity also gives us awesome metrics on our tests and nice statistics on failures and failure rates. Weigh in on this change by joining the discussion, [here](TODO).
```

Comment:
Can also add to the teamcity part, something like:

"teamcity also has nice integration with a large array of test runners, from go tests to karma and sharness. It will give us more detailed feedback about our test runs, even when successful"